src/registry.json

```
@@ -3567,6 +3567,29 @@
         }
       }
     },
+    "npm:@web3-name-sdk/snap": {
+      "id": "npm:@web3-name-sdk/snap",
+      "metadata": {
+        "name": "SPACE ID Web3 Domain ",
```

There's one too many SPACEs here. 😅

```suggestion
        "name": "SPACE ID Web3 Domain",
```